### PR TITLE
Implement commenting block producers

### PR DESCRIPTION
**Pull request by dmytrostriletskyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,6 +186,36 @@ $ curl -X POST \
...
+}
+```
+
+##### Known errors
+
+| Argument  | Level                      | Error message                                               | Status code |
+| :-------: | :------------------------: | ----------------------------------------------------------- | :---------: |
+| -         | General execution          | User with specified e-mail address does not exist.          | 400         |
+| -         | General execution          | Block producer with specified identifier does not exist.    | 400         |
+| -         | General execution          | Block producer with specified identifier does not exist.    | 400         |
```
Comment (anastasiia-bilova) [inline README.md]: Remove the repeated error.

File: directory/block_producer/tests/views/test_comment.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            HTTP_AUTHORIZATION='JWT ' + self.user_token,
+            content_type='application/json',
+        )
+
+        assert expected_result == response.json()
+        assert HTTPStatus.BAD_REQUEST == response.status_code
+
+    def test_comment_block_producer_without_data(self):
+        """
+        Case: to comment a non-exiting block producer without text.
```
Comment (anastasiia-bilova) [inline directory/block_producer/tests/views/test_comment.py]:
```suggestion
        Case: to comment a block producer without text.
```

File: directory/block_producer/views/comment.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            return JsonResponse({'errors': form.errors}, status=HTTPStatus.BAD_REQUEST)
+
+        try:
+            CommentBlockProducer(
+                user=self.user, block_producer=self.block_producer, block_producer_comment=self.block_producer_comment,
+            ).do(
+                user_email=email, block_producer_id=block_producer_id, text=text,
+            )
+        except (
+                BlockProducerWithSpecifiedIdentifierDoesNotExistError,
```
Comment (anastasiia-bilova) [inline directory/block_producer/views/comment.py]:
```suggestion
            BlockProducerWithSpecifiedIdentifierDoesNotExistError,
```

File: README.md (hunk 1 above)
Comment (dmytrostriletskyi) [reply README.md]: @anastasiia-bilova, done.

File: directory/block_producer/tests/views/test_comment.py (hunk 2 above)
Comment (dmytrostriletskyi) [reply directory/block_producer/tests/views/test_comment.py]: @anastasiia-bilova, done.

File: directory/block_producer/views/comment.py (hunk 3 above)
Comment (dmytrostriletskyi) [reply directory/block_producer/views/comment.py]: @anastasiia-bilova, done.

**Pull request merged**